docs/reference/generators.md

```diff
@@ -1148,62 +1145,42 @@ targets:
 
 ***
 
-# NSXT
+## NSXt
```

Comment:
Why this change?

Reply:
Most cases I see have all letters capitalized. https://www.vmware.com/products/nsx/migrate-to-nsx-t.html

Reply:
You're right. Do you mind a dash between NSX and T/v?

Reply:
I don't mind a dash in documentation and comments. Don't change any modules, classes, or method names.